packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -182,29 +188,53 @@ class ReorderableListView extends StatefulWidget {
   final ReorderItemProxyDecorator? proxyDecorator;
 
   /// If true: on desktop platforms, a drag handle is stacked over the
-  /// center of each item's trailing edge; on mobile platforms, a long
-  /// press anywhere on the item starts a drag.
+  /// center of each item's trailing edge.
   ///
   /// The default desktop drag handle is just an [Icons.drag_handle]
-  /// wrapped by a [ReorderableDragStartListener]. On mobile
-  /// platforms, the entire item is wrapped with a
-  /// [ReorderableDelayedDragStartListener].
+  /// wrapped by a [ReorderableDragStartListener].
   ///
   /// To change the appearance or the layout of the drag handles, make
   /// this parameter false and wrap each list item, or a widget within
   /// each list item, with [ReorderableDragStartListener] or
   /// [ReorderableDelayedDragStartListener], or a custom subclass
   /// of [ReorderableDragStartListener].
   ///
-  /// The following sample specifies `buildDefaultDragHandles: false`, and
-  /// uses a [Card] at the leading edge of each item for the item's drag handle.
+  /// The following sample specifies `buildDefaultDragHandlesOnDesktop: false`,
+  /// and uses a [Card] at the leading edge of each item for the item's drag
+  /// handle.
   ///
   /// {@tool dartpad}
   ///
   ///
   /// ** See code in examples/api/lib/material/reorderable_list/reorderable_list_view.build_default_drag_handles.0.dart **
   ///{@end-tool}
-  final bool buildDefaultDragHandles;
+  final bool buildDefaultDragHandlesOnDesktop;
```

Comment:
Removing the existing `buildDefaultDragHandles` is a hard breaking change. Apps and tests currently depend on it. We would need to follow the https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes process to do this.

That said, I am not sure this is the right thing to do here. What was the reason to break this up into two flags? The goal of `buildDefaultDragHandles` was to provide the default handles for a given platform. If we break this up into multiple flags, what happens when we support a new platform that is neither mobile or desktop?

If applications don't like the defaults that we have chosen for a given platform, they can turn off this flag and wrap their list items with what ever drag handle they want.